decidim-templates/db/migrate/20221006184905_migrate_templateable.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class MigrateTemplateable < ActiveRecord::Migration[6.0]
```

Comment:
Sorry I wasn't clear enough but could you also move this to the same migration as the others please?

You can do one-directionnal migrations within the `change` method too using the `reversible` hook as in here:
https://github.com/decidim/decidim/blob/c6958e4a8be7941755f620824d024a20fb95fe44/decidim-core/db/migrate/20220518094535_add_previous_passwords_to_users.rb#L12-L18